src/sections/Home/Layer5-statement/statement.style.js

```diff
@@ -110,7 +110,7 @@ const BannerSectionWrapper = styled.section`
      }
      @media only screen and (max-width: 767px) {
          .hero {
-            display: none;
+            display: none !important;
```

Comment:
Why do we need `!important` here?

Reply:
it is not taking it without `!important`, it tried on `gatsby-image-wrapper` but still not able to hide it on the smaller screens, any suggestions?

Reply:
When I try it via `dev-tools` on the browser, it works without the `! important` 🤔 . Can you update the code to remove it and make the commit, so that I could check it in the preview and see if it's actually working or not